src/hooks/useFocusIn/index.tsx

```diff
@@ -24,15 +24,15 @@ export function useFocusIn(el: RefObject<any>, delay = 3000) {
       }, delay);
     };
 
-    el.current.addEventListener('focusin', onFocusIn);
-    el.current.addEventListener('focusout', onFocusOut);
+    el.current?.addEventListener('focusin', onFocusIn);
```

Comment:
Is this needed? Given that we already return in line 11. I think this is just needed for the removeEventListener

Reply:
Yeah right not needed thanks for the catch!